apps/client/src/components/Nav.test.js

```diff
@@ -60,13 +67,13 @@ it("Nav should render with prop values and should fire events", () => {
     </Wrapper>
   );
 
-  expect(screen.getByText("Prev")).toBeTruthy();
-  expect(screen.getByText("Next")).toBeTruthy();
-  expect(screen.getByText("Next")).toHaveStyle("margin-right: 10px");
+  expect(getByText("Prev")).toBeInTheDocument();
+  expect(getByText("Next")).toBeInTheDocument();
+  expect(getByText("Next")).toHaveStyle("margin-right: 10px");
 
-  fireEvent.click(screen.getByText("Prev"));
+  fireEvent.click(getByText("Prev"));
```

Comment:
Here you actually use previous-button twice, so creating a var could be better :-p

Reply:
Great catch! Fixed